src/admin/navigation/VoteDrawer.jsx

```
@@ -5,7 +5,7 @@ import { useLocation, useNavigate, useParams } from "react-router-dom";
 import { db } from "../../firebase";
 import { DrawerItem } from "./components";
 
-export default function VoteDrawer({}) {
+export default function VoteDrawer({ onClose = () => {} }) {
```

ESLint

[error] 'onClose' is missing in props validation

(react/prop-types)